src/ensembl/src/content/app/browser/track-panel/track-panel-bar/TrackPanelBar.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import { trackPanelBarConfig, TrackPanelBarItem } from './trackPanelBarConfig';
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { Status } from 'src/shared/types/status';
```

Comment:
Since it's a type, move this import down, before styles.

Reply:
✔️